src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.targets

```diff
@@ -524,11 +524,6 @@ Copyright (c) .NET Foundation. All rights reserved.
                                     Value="$(VerifyDependencyInjectionOpenGenericServiceTrimmability)"
                                     Trim="true" />
 
-    <RuntimeHostConfigurationOption Include="System.AggressiveAttributeTrimming"
-                                    Condition="'$(_AggressiveAttributeTrimming)' != ''"
-                                    Value="$(_AggressiveAttributeTrimming)"
-                                    Trim="true" />
-
```

Comment:
How are things structured in the SDK - if we remove it here, would it also be unavailable when building a .NET 8 app using a .NET 10 SDK?